src/Test/modellike.jl

```diff
@@ -344,7 +344,7 @@ function orderedindicestest(model::MOI.ModelLike)
     @test MOI.get(model, MOI.ListOfConstraintIndices{MOI.SingleVariable, MOI.LessThan{Float64}}()) == [c2, c3]
 end
 
-# Test that `MOI.ScalarFunctionConstantNotZero` is thrown when constraint with
+# Test that `MOI.ScalarFunctionConstantNotZero` is thrown when a constraint with
 # a function with nonzero constant is added
```

Comment:
Clarify that this test is optional because solvers could choose to support scalar functions with nonzero constants.